src/Azure/AzureAppServices.HostingStartup/src/PublicAPI.Shipped.txt

```diff
@@ -1,4 +1,5 @@
 #nullable enable
+~Microsoft.AspNetCore.AzureAppServices.HostingStartup.AzureAppServicesHostingStartup.Configure(Microsoft.AspNetCore.Hosting.IWebHostBuilder builder) -> void
```

Comment:
Something seems off here (and the same pattern is in other shipped.txt files)

Reply:
I don't see a problem but it does appear confusing. Two things are going on:

1. PublicAPI.Shipped.txt files are sorted differently because the script uses `Sort-Item -Unique` and the fixer uses a strange sort algorithm (@sharwell @tmat ❔) i.e. this line was previously a couple of lines down
2. PublicAPI.Unshipped.txt files are all now empty. Their content is mixed into the PublicAPI.Shipped.txt files i.e. the last line was previously in the PublicAPI.Unshipped.txt file.

Reply:
> ~Microsoft.AspNetCore.AzureAppServices.HostingStartup.AzureAppServicesHostingStartup.Configure(Microsoft.AspNetCore.Hosting.IWebHostBuilder builder) -> void

> Microsoft.AspNetCore.AzureAppServices.HostingStartup.AzureAppServicesHostingStartup.Configure(Microsoft.AspNetCore.Hosting.IWebHostBuilder! builder) -> void

These are the same except one has nullable annotations.

Reply:
Hmm, not sure how we avoided the `*REMOVED*` requirement. I'll delete this line and fix up another PublicAPI.Shipped.txt file that lost its line endings. Have a couple of larger files that are now just one line long ☹️

This oddity doesn't seem to matter much but feel free to point out other instances if you notice any.

```suggestion
```